dashboard/modules/reporter/reporter_agent.py

```diff
@@ -154,7 +154,30 @@ def jsonify_asdict(o) -> str:
         "raylet_cpu", "CPU usage of the raylet on a node.", "percentage", ["ip", "pid"]
     ),
     "raylet_mem": Gauge(
-        "raylet_mem", "Memory usage of the raylet on a node", "mb", ["ip", "pid"]
+        "raylet_mem",
+        "Memory usage of the raylet on a node. Type includes uss and the remaining rss "
+        "on Linux, and just rss on other platforms.",
+        "MB",
+        ["ip", "pid", "mem_type"],
+    ),
+    "workers_count": Gauge(
```

Comment:
I think it would be more suitable to report this directly from the raylet, since it has more visibility into the types of workers, etc (actually I didn't realize we weren't already reporting this!). The raylet already reports [total number of workers started](https://github.com/ray-project/ray/blob/master/src/ray/stats/metric_defs.h#L200), just not current number of workers.

Reply:
What about also reporting worker count by type from Raylet? I think it could be useful to have an outside process monitoring the state. The cost should be small. Or I can remove `workers_count` here.

Reply:
Hmm, I don't see why it's useful to have the outside process monitoring this state compared to the raylet, even if the cost is small. Wouldn't it be better to only report it one place and to be able to have more visibility into the worker type?

Reply:
Monitoring outside of Raylet may be useful to check if there are leaked / zombie workers. Removed `workers_count` for now.